x/authz/exported/authorizations.go

```diff
@@ -10,14 +10,27 @@ import (
 type Authorization interface {
```

Comment:
Can we move this out of the `exported` package?

Reply:
yes, we should. I forgot to put it in the task list.